demo/docker-compose.yaml

```diff
@@ -25,11 +25,11 @@ services:
       - 12800 # HTTP
     networks:
       - manual
-    environment:
-      SW_KAFKA_FETCHER: default
-      SW_KAFKA_FETCHER_SERVERS: kafka:9092
-      SW_KAFKA_FETCHER_PARTITIONS: 2
-      SW_KAFKA_FETCHER_PARTITIONS_FACTOR: 1
+#    environment:
+#      SW_KAFKA_FETCHER: default
+#      SW_KAFKA_FETCHER_SERVERS: kafka:9092
+#      SW_KAFKA_FETCHER_PARTITIONS: 2
+#      SW_KAFKA_FETCHER_PARTITIONS_FACTOR: 1
```

Comment:
Remove these?

Reply:
I intentionally left them here for developers to try out the agent, and since kafka is not the default so I commented them out. Should I simply uncomment them?

Reply:
Yes uncommenting them is preferable to me